src/cascadia/WindowsTerminal/AppHost.h

```diff
@@ -25,6 +25,8 @@ class AppHost
     winrt::TerminalApp::App _app;
     winrt::TerminalApp::AppLogic _logic;
 
+    void _handleCommandlineArgs();

```

Comment:
So I figured that new methods and such should be this way, following from #721. I suppose since this class is _PascalCase, I'll change this one back, but leave the methods I've added in new classes

Reply:
They should be, probably, yeah. But let's change them together and not mix in a class.